student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [Optional] I think we are only inheriting the properties hence you still need to default their values: That's

`def initialize(age, classroom, name = "Unknown", parent_permission = true)`


- Same goes for the Teacher's class

Reply:
Thank you @FrederickMih. I will take care of these